core/runtime/binaryen/runtime_manager.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime_manager.hpp"
+
+#include <binaryen/wasm-binary.h>
+
+#include <crypto/hasher/hasher_impl.hpp>
+
+#include "runtime_external_interface.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::runtime::binaryen,
+                            RuntimeManager::Error,
+                            e) {
+  using kagome::runtime::binaryen::RuntimeManager;
```

Comment:
```suggestion
  using E = kagome::runtime::binaryen::RuntimeManager::Error;
```
You can do this and in cases just write E::EMPTY_STATE_CODE, etc.
Just looks a bit neater IMHO.